orienteer-camel/src/main/java/org/orienteer/camel/Module.java

```diff
@@ -45,6 +46,8 @@ public void onInitialize(OrienteerWebApplication app, ODatabaseDocument db) {
 		app.setMetaData(CamelWidget.INTEGRATION_SESSIONS_KEY, new ConcurrentHashMap<String,CamelContext>());
 		app.mountPages("org.orienteer.camel.web");
 		app.registerWidgets("org.orienteer.camel.widget");
+		MethodManager.get().addModule(Module.class);
```

Comment:
Can we have entry point for registration on app? And method "addModule" confusing here: modules are already installed - lets do not invent modules for different stuff.